lib/pages/edit_page/bloc/edit_page_bloc.dart

```diff
@@ -8,15 +8,15 @@ part 'edit_page_event.dart';
 part 'edit_page_state.dart';
 
 class EditPageBloc extends Bloc<EditPageEvent, EditPageState> {
-  EditPageBloc({required this.storage}) : super(const EditPageState()) {
+  EditPageBloc(Connection connection, {required this.storage})
+      : super(EditPageState.connection(connection)) {
```

Comment:
Maybe better remove constructor and use previous constructor?
```suggestion
  EditPageBloc(Connection connection, {required this.storage})
      : super(EditPageState(
            name: connection.name,
            accessKey: connection.accessKey,
            secretKey: connection.secretKey,
            bucket: connection.bucket,
            endpoint: connection.endpoint)) {
```

What do you think?